libraries/AP_Airspeed/AP_Airspeed.cpp

```diff
@@ -457,6 +512,16 @@ void AP_Airspeed::update(bool log)
 
     check_sensor_failures();
 
+    // update vel_dot at 50Hz
+    counter++;
```

Comment:
No counters! Delta time only please, this doesn't support variations in scheduling, or platforms that run these at different update rates.

Reply:
microsecond or millisecond?

Reply:
If a counter worked I'd guess millis is fine.

Reply:
If airspeed is only called at 10Hz, it's going to be hard to calculate vel_dot at 50Hz...